lib/node_loader.js

```diff
@@ -59,12 +61,9 @@ AWS.CredentialProviderChain.defaultProviders = [
   function () { return new AWS.EnvironmentCredentials('AWS'); },
   function () { return new AWS.EnvironmentCredentials('AMAZON'); },
   function () { return new AWS.SharedIniFileCredentials(); },
-  function () {
-    if (AWS.ECSCredentials.prototype.isConfiguredForEcsCredentials()) {
```

Comment:
I don't see the diff for `EcsCredentials`. Do I miss it?

Reply:
ECSCredentials already throws an error if ENV_RELATIVE_URI or ENV_FULL_URI aren't set. isConfiguredForEcsCredentials could be removed if ECSCredentials is going to be put into the default providers.